src/anbox/config.h.in

```diff
@@ -44,10 +43,14 @@ class SystemConfiguration {
   std::string resource_dir() const;
 
  protected:
-  SystemConfiguration() = default;
```

Comment:
Lets move SystemConfiguration into it's own file system_configuration.cpp/.h and have a dedicated config.h